crud.rb

```diff
@@ -15,7 +16,21 @@ def initialize
 # redirects '/' route to '/users' route (DONE)
 #-----------------------------------------------------------------
 get '/' do
-  redirect '/users'
+  redirect '/homepage'
+end
+
+# Displays homepage for users to log in or sign up
+#-----------------------------------------------------------------
+get '/homepage' do
+  erb :"homepage/index"
+end
+
+# Logs out user
+#-----------------------------------------------------------------
+get '/users/logout' do
```

Comment:
Use a `post` here since `get` requests are supposed to not have any side-effects.